packages/pocket-ic/tests/slow.rs

```diff
@@ -1,151 +1,20 @@
-use candid::Principal;
+use candid::{Encode, Principal};
 use pocket_ic::{common::rest::DtsFlag, PocketIc, PocketIcBuilder, UserError, WasmResult};
-use std::{thread, time::Duration};
+use std::time::Duration;
 
 // 200T cycles
 const INIT_CYCLES: u128 = 200_000_000_000_000;
 
-// Canister code incrementing a counter in every heartbeat
-// and exporting a query method to read the counter.
-const AUTO_PROGRESS_WAT: &str = r#"
-    (module
-        (import "ic0" "msg_reply" (func $msg_reply))
-        (import "ic0" "msg_reply_data_append"
-            (func $msg_reply_data_append (param i32 i32)))
-        (func $inc
-            ;; Increment a counter.
-            (i32.store
-                (i32.const 0)
-                (i32.add (i32.load (i32.const 0)) (i32.const 1))))
-        (func $read
-            (call $msg_reply_data_append
-                (i32.const 0) ;; the counter from heap[0]
-                (i32.const 4)) ;; length
-            (call $msg_reply))
-        (memory $memory 1)
-        (export "canister_query read" (func $read))
-        (export "canister_heartbeat" (func $inc))
-    )
-"#;
-
-#[test]
-fn test_auto_progress() {
```

Comment:
This test has been moved to `//rs/pocket_ic_server:test` where we can easily read the server's stdout.